### ISSUE TITLE
Mismatch with default config when filtering by parameter name 

### ISSUE THREAD

**Issue by ouvtk** [jvoisin/snuffleupagus#407]
Hello everyone,

First of all want to thank you for making this tool available, I bet it already saved and will save even more people from headaches.
But as it usually happens, I never said it here before until I had something less positive to say.

We run PHP 7.4 on Alpine Linux, and after update of snuffleupagus to 0.7.1 warnings appeared, which were not the case in the previous versions.

For example:
```
Warning: [snuffleupagus][0.0.0.0][config][log] It seems that you are filtering on a parameter 'command' of the function 'shell_exec', but the parameter does not exists. in php shell code on line 1
Warning: [snuffleupagus][0.0.0.0][config][log]   - 0 parameter's name: 'cmd' in php shell code on line 1

Warning: [snuffleupagus][0.0.0.0][config][log] It seems that you are filtering on a parameter 'var_array' of the function 'extract', but the parameter does not exists. in php shell code on line 1
Warning: [snuffleupagus][0.0.0.0][config][log]   - 0 parameter's name: 'arg' in php shell code on line 1
Warning: [snuffleupagus][0.0.0.0][config][log]   - 1 parameter's name: 'extract_type' in php shell code on line 1
Warning: [snuffleupagus][0.0.0.0][config][log]   - 2 parameter's name: 'prefix' in php shell code on line 1
```

Looks like related to changes in:
 - #367
 - #379

Steps to reproduce:

1. Create Dockerfile:
   ```
   FROM php:7.4-alpine
   RUN apk add --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/testing php7-snuffleupagus
   RUN echo -e 'extension=/usr/lib/php7/modules/snuffleupagus.so\nsp.configuration_file=/etc/php7/conf.d/snuffleupagus.rules' \
       > '/usr/local/etc/php/conf.d/php7-snuffleupagus.ini'
   ```

2. Build the container: `docker build -t snuffleupagus-test .`
3. Run the PHP interactive shell: `docker run -i snuffleupagus-test php -a`
4. Type `shell_exec('ls');` or `extract($a = []);`. We didn't encounter any other problematic functions.

Please point me into the right direction:
 - Is the new behaviour a correct one?
 - Does configuration file need to be updated?

And how can I assist by looking into this issue, if it is an issue

Thank you and stay safe

**Comment by jvoisin** [comment]
Sigh, why does php keep changing functions parameter names between minor releases :/

Any, it should be mitigated/fixed by baff5856be20579908497dba99e1e3eb20080684

Thanks for your report, and your patience.

**Comment by ouvtk** [comment]
@jvoisin Perfect, thank you for following up on this 🙇
